reference/5.1/Microsoft.PowerShell.Core/About/about_Special_Characters.md

```diff
@@ -146,16 +146,32 @@ Column1         Column2         Column3
 
 ## Vertical tab (`v)
 
-The horizontal tab (`` `v ``) character advances to the next vertical tab stop
-and writes the remaining output at that point. This has no effect in the
-default Windows console.
+The horizontal tabs (```v``) effect is device and terminal dependant. Here
+are _some_ examples from different terminals:
 
 ```powershell
 Write-Host "There is a vertical tab`vbetween the words."
 ```
 
-The following example shows the output you would get on a printer or in a
-different console host.
+The following example shows the output of a default terminal in Windows. The
+terminal interprets ```v`` as the `♂` character with no extra spacing added.
+
+```Output
+There is a vertical tab♂between the words.
+```
+
+The following example shows the output of [Windows Terminal](https://www.microsoft.com/p/windows-terminal/9n0dx20hk701).
+The ```v`` character acts as a carriage return character and the rest of the
+output is printed on the following line.
+
+```Output
+There is a vertical tab
+between the words.
+```
+
+The following example shows the output you would get on a printer or in a unix
+based console host. The vertical tab (```v``) character advances to the next
+vertical tab stop and writes the remaining output at that point.
```

Comment:
```suggestion
The following examples show the rendered output of the vertical tab in some common environments.

The Windows Console host application interprets (`` `v ``)  as a special character with no extra spacing added.

```Output
There is a vertical tab♂between the words.
```

The [Windows Terminal](https://www.microsoft.com/p/windows-terminal/9n0dx20hk701) renders the vertical tab character as a carriage return and line feed. The rest of the
output is printed at the beginning of the next line.

```Output
There is a vertical tab
between the words.
```

On printers or in a unix-based consoles, the vertical tab character advances to the next
line and writes the remaining output at that point.
```